modules/monitoring/library/Monitoring/Hook/DetailviewExtensionHook.php

```diff
@@ -68,4 +78,34 @@ public function setView($view)
         $this->view = $view;
         return $this;
     }
+
+    /**
+     * Get {@link module}
```

Comment:
Please don't link private/protected properties in the documentation of the public interface. The documentation should read "Get the instance of the module this hook belongs to" or something similar.